public/views/index/body.ejs

```diff
@@ -40,10 +40,13 @@
 
             <div id="home" class="section"<% if(signin) { %> style="display:none;"<% } %>>
                 <div class="inner cover">
-                    <h1 class="cover-heading"><i class="fa fa-file-text"></i> CodiMD</h1>
+                    <h1 class="cover-heading"><img src="<%- serverURL %>/banner/banner_vertical_color.svg" width="400" alt="HedgeDoc logo"></h1>
                     <p class="lead">
                         <%= __('Best way to write and share your knowledge in markdown.') %>
                     </p>
+                    <span class="alert alert-info" style="display: inline-block;">
+                        CodiMD was renamed to HedgeDoc. <a href="https://hedgedoc.github.io/history/" rel="noopener noreferrer" target="_blank" class="text-danger">Read more</a>
```

Comment:
Do we want to point to `hedgedoc.github.io/history/` or to `hedgedoc.org/history/`?